### DISCUSSION TITLE
Future of Bootscore

### DISCUSSION THREAD

**Discussion by kemalkastrataj** [bootscore/bootscore#655] [General]
**What is the future of bootscore?** Will it prevail as a classic theme or is a block based theme the next step, I worked on the pre-loader way back in time and I have a very customized version of bootscore but when looking at future development I would kind of like to see bootscore as a block based theme with bootstrap built in.

**PS:** Great things happening with WP lately; Google Interactivity API (Now part of Gutenberg)

**Comment by justinkruit** [comment]
Great question! We are certainly looking into more support for Gutenberg natively, but probably not via the Gutenberg Site Editor. By default bootScore is mostly meant as the building base for your WordPress theme, but we also want to expand that by bringing parts of it to Gutenberg. 

For now most changes regarding this are still the pipeline, but we hope to bring some more stuff soon!

**Comment by r3fl3x** [comment]
I would be happy if Bootscore wouldn't lose the general idea:

I copy&paste from the website:
There are no Customizer or Backend settings. Basic HTML, CSS/SCSS, and maybe some PHP and JavaScript skills are required for customizing. No drag’n’drop, **just simple code.**

I think that the last three words is the key :)

**Comment by crftwrk** [comment]
I was joining a WP event last week about full site editing. This is a complete different approach and won't fit here, because theme data is stored in the database and not in the files. Think we should go the "Hybrid themes" way to build a bridge between classic and block theme combining the best of two worlds. This will keep Bootscore's spirit/workflow and we can add step by step more support to the block editor, adding a `theme.json` and block templates.

Found a helpful resource https://fullsiteediting.com/lessons/adding-full-site-editing-features-to-classic-themes/ where we can cherry-pick our needs.

First steps are already done in https://github.com/bootscore/bootscore/pull/660, https://github.com/bootscore/bootscore/pull/662 and https://github.com/bootscore/bootscore/pull/670.
